Earthfile

```diff
@@ -64,17 +69,58 @@ lint:
     COPY . .
     RUN golangci-lint run --timeout=3m
 
+DOWNLOAD_BINARIES:
+    COMMAND
+    IF $FIPS_ENABLED
+        RUN curl -L "https://storage.googleapis.com/spectro-fips/cri-tools-${CRICTL_VERSION}.tar.gz" | sudo tar -C /usr/bin/ -xz
+        RUN curl -L --remote-name-all https://storage.googleapis.com/spectro-fips/${KUBEADM_VERSION}/kubeadm
+        RUN curl -L --remote-name-all https://storage.googleapis.com/spectro-fips/${KUBEADM_VERSION}/kubelet
+        RUN curl -L --remote-name-all https://storage.googleapis.com/spectro-fips/${KUBEADM_VERSION}/kubectl
+    ELSE
```

Comment:
@jzhoucliqr, Would it be fine, if we always use FIPS binary inside image or should we tag FIPS images separately in kairos repo?